### PR TITLE
[ISSUE #3252] Field may be 'final'[HttpSummaryMetrics]

### LINKED ISSUE #3252: [Enhancement] Field may be 'final'[HttpSummaryMetrics]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/eventmesh/issues?q=is%3Aissue) and found no similar issues.

### Enhancement Request

image
located at:
eventmesh-metrics-plugin/eventmesh-metrics-api/src/main/java/org/apache/eventmesh/metrics/api/model/HttpSummaryMetrics.java line 46,49,51,53,55,112,123,125,127,130,136,185,187,189,191,193,195,260,262,264,266,268,270,336,340,344

### Describe the solution you'd like

add final.

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by ibrijesh**
Fixes #3252 .

### Modifications

*Describe the modifications you've done.*
  Added final 

### Documentation

- Does this pull request introduce a new feature? (yes / no)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
- If a feature is not applicable for documentation, explain why?
- If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

**Comment by mxsm** [comment]
@ibrijesh welcome to contribute

File: eventmesh-metrics-plugin/eventmesh-metrics-api/src/main/java/org/apache/eventmesh/metrics/api/model/HttpSummaryMetrics.java [hunk 1]
```diff
@@ -120,20 +120,20 @@ public float avgHTTPBodyDecodeCost() {
...
-    private AtomicLong sendBatchMsgFailNumSum = new AtomicLong(0);
+    private final AtomicLong sendBatchMsgFailNumSum = new AtomicLong(0);
 
     // This is a cumulative value
-    private AtomicLong sendBatchMsgDiscardNumSum = new AtomicLong(0);
+    private final AtomicLong sendBatchMsgDiscardNumSum = new AtomicLong(0);
 
     public void recordSendBatchMsgDiscard(long delta) {
         sendBatchMsgDiscardNumSum.addAndGet(delta);
     }
```
Comment (mxsm) [inline eventmesh-metrics-plugin/eventmesh-metrics-api/src/main/java/org/apache/eventmesh/metrics/api/model/HttpSummaryMetrics.java:135]: Adjust the position of variables, move them to the beginning of the file class, and place the methods below the variables.

**Review by mxsm — COMMENTED** [review]
@ibrijesh Adjust the position of variables, move them to the beginning of the file class, and place the methods below the variables.

**Review by mxsm — APPROVED** [review]
LGTM~

**Review by Alonexc — APPROVED** [review]
LGTM

**Review by xwm1992 — APPROVED** [review]

**Pull request merged**